### ISSUE TITLE
drop arbitrary distro specific assumptions in setup.py

### ISSUE THREAD

**Issue by ng-0** [Bitmessage/PyBitmessage#1021]
Distro specific jobs are best left to downstream package maintainers.
The best upstream assumptions in setup files can create is more downstream patching.
You might have good intentions, but it really is not your job to make it right on all systems.
If package maintainers run into something which is generic enough to be included, they
will do this. Everything else is just patching around work and creates duplication which
shouldn't exist in thr first place.

For example if someone would update the guix package, they would notice that the piece about Guix in the setup.py will not work for multiple reasons:

```
    elif os.path.isfile("/etc/config.scm"):
detectOS.result = "Guix"
```

**Comment by PeterSurda** [comment]
Thank you for the report. I'll re-evaluate this prior to 0.6.3 release, based on feedback I'm leaning towards moving distro-specifics to a separate file.

**Comment by PeterSurda** [comment]
Fixed in 196d688b138393d1d540df3322844dfe7e7c02ba.
